server/src/workout/workout-endpoints.ts

```diff
@@ -0,0 +1,25 @@
+import { createWorkoutServer, deleteWorkout, getWorkoutById, updateWorkout } from "./workout-utils";
+import { Request, Response, Application } from "express";
+import { Database } from "sqlite";
+
+export function createWorkoutEndpoints(app: Application, db: Database) {
```

Comment:
I think that we will also need a `GET workouts` endpoint where we get a list of all the workouts in the database.